src/Middleware/NetworkResolver.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+namespace Yiisoft\Yii\Web\Middleware;
+
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Yiisoft\Yii\Web\NetworkResolver\NetworkResolverInterface;
+use Yiisoft\Yii\Web\NetworkResolver\NotTrustedHeadersInterface;
```

Comment:
There's no such class in the pull request.

Reply:
removed